backend/routes/review.js

```diff
@@ -70,9 +70,9 @@ router.post('/:id/delete', async (req, res) => {
         _id: req.params.id
       },
       {
-        $push: {
```

Comment:
Does this change mean that you don't have to do a push request to the database to change the deleted field of the review to true?

Reply:
yeah, because the await command is already set to findOneAndUpdate, it's already assuming to update the given field